randovania/games/prime2/patcher/claris_randomizer.py

```diff
@@ -198,9 +200,9 @@ def apply_patcher_file(game_root: Path,
 
     last_version = get_patch_version(game_root)
     if last_version > CURRENT_PATCH_VERSION:
-        raise ClarisRandomizerExportError(f"Game at {game_root} was last patched with version {last_version}, "
-                            f"which is above supported version {CURRENT_PATCH_VERSION}. "
-                            f"\nPlease press 'Delete internal copy'.", None)
+        raise UnableToExportError(
+            "The internal was outdated and has been deleted. Please export again and select an ISO.",
```

Comment:
"The internal was outadated" reads a bit weird. Probably meant "the internal game copy"?

Reply:
Apparently I forgot to finish the sentence.